src/components/FeatureFlagModal/DevFlagsBox.tsx

```diff
@@ -0,0 +1,64 @@
+import { BaseVariant, featureFlagSettings } from 'featureFlags'
+import { useAtomValue } from 'jotai/utils'
+import { useMemo, useState } from 'react'
+import { useGate } from 'statsig-react'
+import styled from 'styled-components'
+import { ThemedText } from 'theme/components'
+import { isDevelopmentEnv, isStagingEnv } from 'utils/env'
+
+const Box = styled.div`
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+  background-color: ${({ theme }) => theme.surface1};
+  padding: 10px;
+  border: 1px solid ${({ theme }) => theme.accent1};
+  z-index: 1000;
+  width: 300px;
+`
+const TopBar = styled.div`
```

Comment:
The top bar adds a margin which makes the box look unbalanced when it is collapsed.